pkg/kubelet/kubelet.go

```diff
@@ -1469,7 +1469,10 @@ func (kl *Kubelet) Run(updates <-chan kubetypes.PodUpdate) {
 
 	if kl.kubeClient != nil {
 		// Start syncing node status immediately, this may set up things the runtime needs to run.
```

Comment:
this comment is no longer accurate with the jitter?

Reply:
Indeed [it wasn't before this PR too :) ]